src/main/java/org/terracotta/build/plugins/docker/DockerBuildExtension.java

```diff
@@ -30,6 +30,8 @@ public void contents(Action<CopySpec> action) {
 
   public abstract RegularFileProperty getDockerFile();
 
+  public abstract MapProperty<String, String> getDockerEnv();
```

Comment:
> Right now this is only getting applied to one specific `DockerBuild` instance. If we're going to have this exposed in the build extension I think it would make more sense for anything added here to be applied to _all_ the `DockerTask` instances.

I think this is what we settled on yesterday and it might be useful also when switching to buildx after.

> That said I'm not sure I'd worry about putting this here. I think it's fine to just add the property to `DockerTask` and let the convention plugin call:
> 
> ```groovy
> tasks.withType(DockerTask) {
>   dockerEnv += ...
> }
> ```

ok will try

Reply:
it works ;-) Updated the EE Pr too.